cloudwatch_monitoring/lookups.py

```diff
@@ -341,6 +341,64 @@
     }
 }
 
+# sns topics
```

Comment:
I'm not sure we care about all of these.... :thinking: 

I'm concerned that we're filling the dashboards with so much information that they're not useful.

Reply:
@ayan-usgs I agree that the dash is actually getting a bit bloated.  I think we've reached a point where it could be worthwhile to consider removing some stuff, or splitting things out into other dashboards.